src/libs/ui/docsetlistitemdelegate.cpp

```diff
@@ -59,13 +59,19 @@ void DocsetListItemDelegate::paint(QPainter *painter,
     font.setItalic(true);
 
     const QFontMetrics fontMetrics(font);
+    const int margin = 4; // Random small number
```

Comment:
There's probably margin provided by theme we should use here. I'll look into this later.